app/views/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_with model: @post, url: user_posts_path, method: :post, local: true, scope: :new_post do |form| %>
```

Comment:
- [ ] I would like to recommend you create the form for creating posts inside the post view, then you will get **views/posts/new.html.erb** , For each controller, it is very important to associate the directory in the app/views directory which holds the template files that make up the views associated with that controller to display the view that results from each controller action. Please fix it 👍

Reply:
@codecaiine thank you for the code review I really do appreciate it, and I hope I've met the required changes above to complete this milestone